src/components/home/partners/SecondaryPartners.jsx

```diff
@@ -0,0 +1,84 @@
+import Box from '@mui/material/Box';
+import Container from '@mui/material/Container';
+import Grid from '@mui/material/Grid';
+import Typography from '@mui/material/Typography';
+
+import { secondaryPartnerList } from './secondaryPartnerList';
+
+const partnerGridStyle = {
+  xs: 12,
+  md: 6,
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+  flexDirection: 'column',
+  pt: { xs: '40px', md: 0 }
+};
+
+const displaySecondaryPartners = ({
+  index,
+  company,
+  testimonial,
+  testimonialTwo,
+  testimonialAuthorTwo,
+  testimonialAuthor,
+  partnerLogo,
+  website
+}) => {
+  const gradientStyle =
+    index % 2 === 0
+      ? 'linear-gradient(90deg, rgba(217, 217, 217, 0) 38.54%, rgba(217, 217, 217, 0.4) 82.29%)'
+      : 'linear-gradient(270deg, rgba(217, 217, 217, 0) 38.54%, rgba(217, 217, 217, 0.4) 82.29%)';
+
```

Comment:
Empty line not needed here, but it doesn't matter

Reply:
removed in next push